diff -p -up ./libbrasero-burn/Makefile.am.orig ./libbrasero-burn/Makefile.am
--- ./libbrasero-burn/Makefile.am.orig	2012-02-25 07:07:01.547428714 +0000
+++ ./libbrasero-burn/Makefile.am	2012-02-25 07:07:23.641304204 +0000
@@ -67,7 +67,8 @@ libbrasero_burn3_la_LIBADD =					\
 	$(BRASERO_GTK_LIBS)					\
 	$(BRASERO_GSTREAMER_LIBS)				\
 	$(BRASERO_LIBNOTIFY_LIBS)					\
-	$(BRASERO_CANBERRA_LIBS)
+	$(BRASERO_CANBERRA_LIBS)				\
+	-lm
 
 libbrasero_burn3_la_LDFLAGS =					\
 	-version-info $(LIBBRASERO_LT_VERSION)			\
